### ISSUE TITLE
litemono-8m accuracy

### ISSUE THREAD

**Issue by rnlee1998** [noahzn/Lite-Mono#51]
I appreciate your work very much, and I have reproduced your code. The **litemono** model I trained is similar to the reported result, but the **litemono-8m** is worse than your reported. What is the reason? If possible, can you provide your training environment?

my litemono-8m result:
       abs_rel |   sq_rel |     rmse | rmse_log |       a1 |       a2 |       a3 | 
&   0.105  &   0.847  &   4.614  &   0.182  &   0.893  &   0.963  &   0.982

**Comment by noahzn** [comment]
Hi, thank you for your interest in our work.

It seems your training converged to a local minimum. I got similar results both using TITAN Xp (CUDA11.0, PyTorch 1.7.1) and NVIDIA A40 (CUDA 11.3, PyTorch 1.12.1).
You can also try to set `--lr` as `[0.0001, 5e-6, 16, 0.0001, 1e-5, 16]`, then the learning rate will be reset at the 16th epoch. If you do this the best result might be achieved at epoch 16 or 17. So, please check all the epochs.

**Comment by rnlee1998** [comment]
Thanks for your reply and I wander the python=3.8 or 3.9 ?

**Comment by noahzn** [comment]
I used Python 3.7.4 and 3.8.13, respectively.

**Comment by noahzn** [comment]
I'm now closing this issue due to no response. Please feel free to reopen it if you still have questions.
